packages/e2e-test-utils/src/get-list-view-blocks.js

```diff
@@ -6,6 +6,6 @@
  */
 export async function getListViewBlocks( blockLabel ) {
 	return page.$x(
-		`//table[contains(@aria-label,'Block navigation structure')]//a[span[text()='${ blockLabel }']]`
+		`//table[contains(@aria-label,'Block navigation structure')]//a//span[text()='${ blockLabel }']`
```

Comment:
This change means that it looks for `span` as any child of the `a` tag instead of a direct child. This allows us to adjust the level of nesting between the `a` tag and the `blockLabel` text without breaking tests that depend on this function (🤞)

Reply:
I see I've missed a couple of tests that also use XPaths expecting a particular sibling of the label. E.g.

* https://github.com/WordPress/gutenberg/blob/b07ea2cf7d17b4f12855fce8d9baa36d86c803a6/packages/e2e-tests/specs/editor/blocks/navigation.test.js#L1054 and
* https://github.com/WordPress/gutenberg/blob/a92b73ceb9e58dc600fb16f2865b2716ee67cef0/packages/e2e-tests/specs/editor/blocks/gallery.test.js#L106

I'll fix them up tomorrow

Reply:
Resolved in https://github.com/WordPress/gutenberg/commit/7344bed5fbc6c469264fea9bd9e7941933a81b00